src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -930,6 +974,12 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
 
         if (ptr.PointerDeviceType() == Windows::Devices::Input::PointerDeviceType::Mouse || ptr.PointerDeviceType() == Windows::Devices::Input::PointerDeviceType::Pen)
         {
+            if (_TrySendMouseEvent(point, true))

```

Comment:
Do you mean to send `false` instead? At this point in `_PointerMovedHandler` a mouse button may or may not be held down.

Reply:
Just checked. We need to send it as down, not up. I'll update the comment in the method header though.